wormhole-connect/src/store/transferInput.ts

```diff
@@ -104,6 +104,7 @@ export type RouteState = {
   name: string;
   supported: boolean;
   available: boolean;
+  availabilityError: string;
```

Comment:
I know empty strings are "falsy" but in general I like being explicit when something is optional and using `undefined` rather than an empty string when there is no error.

```suggestion
  availabilityError?: string;
```

Reply:
Fair enough.